src/vis/sidebar/CloseButton.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export function CloseButton({ closeCallback }: { closeCallback: () => void }) {
+  return (
+    <div className="position-absolute end-0 top-0">
+      <button onClick={() => closeCallback()} className="btn btn-primary-outline" type="button">
+        <i className="fas fa-times" />
```

Comment:
Please replace with [Bootstrap close button](https://getbootstrap.com/docs/5.1/components/close-button/) that we use everywhere.

Reply:
done